src/main/java/com/buddy/tutorial/controller/NewsApiController.java

```diff
@@ -0,0 +1,39 @@
+package com.buddy.tutorial.controller;
+
+import com.buddy.tutorial.model.ResponseModel;
+import com.buddy.tutorial.model.TopHeadlinesAPIResponse;
+import com.buddy.tutorial.model.TopHeadlinesDetail;
+import com.buddy.tutorial.service.NewsApiService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class NewsApiController {
+
+
+    private static final Integer STATUS_CODE_OK = 200;
+    @Autowired
+    private NewsApiService newsApiService;
+
+    @GetMapping("top-headlines")
+    public ResponseModel getTopHeadlines(@RequestParam(defaultValue = "en") String language, @RequestParam(required = false) String category) {
```

Comment:
As per the requirement, We should only accept/work with two languages. English and Arabic. But i can pass any value here. Also how do the user knows about the possible values?
Same applies with category field as well